package.json

```diff
@@ -5,7 +5,7 @@
 , "repository": "goto-bus-stop/xkcd-37"
 , "author": "René Kooi <rene@kooi.me>"
 , "scripts":
-  { "test": "standard && node ./test.js" }
+  { "test": "standard index.js && node ./test.js" }
```

Comment:
You can disable that error for that line alone by adding an eslint comment at the end of the offending line:

```js
// eslint-disable-line no-useless-escape
```